source/src/protocols/cyclic_peptide_predict/SimpleCycpepPredictApplication.cc

```diff
@@ -3232,6 +3374,17 @@ SimpleCycpepPredictApplication::add_closebond_logic_thioether_lariat(
 	// I think the above is strictly unnecessary; no one needs to randomize the un-involved sidechain
 	// and I'm going to say that we basically cannot randomize any dihedrals in 2-chloro-acetyl.
 	// In theory you could randomize CONN2-CP2-CO-N and CO-N-CA-C but that's tomorrow's problem. AMW TODO
+
+	//This application appears to only produce cis CA-N-CO-CP2 bonds without randomization,
+	// trans should be energetically favored, so setting bond to trans here
+	genkic->add_perturber( protocols::generalized_kinematic_closure::perturber::set_dihedral );
+	utility::vector1< core::id::NamedAtomID > omega_vect;
+	omega_vect.push_back( core::id::NamedAtomID( "N", cyclization_point_end));
+	omega_vect.push_back( core::id::NamedAtomID( "CO", cyclization_point_end));
+	genkic->add_atomset_to_perturber_atomset_list( omega_vect );
+	genkic->add_value_to_perturber_value_list( 180.0 );
```

Comment:
Does this addition change the behavior of any current protocols? Is that acceptable in non-lanthionine cases?

Reply:
This changes the behavior of the thioether lariat. It was predicting the N terminal peptide bond to always be cis. This changes it to be trans, which causes the test difference. I messaged @vmullig about this but have not heard back yet.

Reply:
Do we have good experimental data on the cis/trans prevalence in thioether lariats?

Reply:
I am not sure how much data there is. I know that in this structure (https://www.rcsb.org/structure/8FEG), the thioether is conjugated to a trans peptide bond.